railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults.rb.tt

```diff
@@ -1,37 +1,34 @@
 # Be sure to restart your server when you modify this file.
-# This file contains all the new default configuration options from
-# Rails 5.0.
-<%- unless options[:skip_active_record] -%>
-
-# Require `belongs_to` associations by default. This is a new Rails 5.0
-# default, so it is introduced as a configuration option to ensure that apps
-# made on earlier versions of Rails are not affected when upgrading.
-Rails.application.config.active_record.belongs_to_required_by_default = true
+#
+# This file contains migration options to ease your Rails 5.0 upgrade.
+# They will be removed in the next major Rails version.
```

Comment:
This isn't necessarily true. We don't need to mention this here, each of the setters will have a deprecation warning individually before they are removed.